app/services/idv/session.rb

```diff
@@ -63,6 +63,15 @@ def respond_to_missing?(method_sym, include_private)
     end
 
     def create_profile_from_applicant_with_password(user_password, is_enhanced_ipp)
+      if current_user.has_in_person_enrollment?
+        UspsInPersonProofing::EnrollmentHelper.schedule_in_person_enrollment(
```

Comment:
Unimportant nit: I think these two lines read kind of confusingly: "If the user already as an in-person enrollment, schedule an in-person enrollment for them." I'm not sure if there's an easy way to fix this with a rename, though?

Reply:
I also find this confusing.  Maybe we could rename `has_in_person_enrollment?` to `has_unscheduled_in_person_enrollment?`

Reply:
I agree with that the naming could be better... my only issue is that the `has_in_person_enrollment?` checks for `pending` or `establishing` in-person enrollments. Unscheduled should only care about `establishing` in-person enrollments right?

Reply:
Would `needs_in_person_verification?` be a clearer method name?  I got that idea from the `in_person_verification_needed` param for profile_maker’s [save_profile]( https://github.com/18F/identity-idp/blob/main/app/services/idv/session.rb#L70) method. 

It could also be renamed `has_begun_in_person_proofing?`, but I worry that’s too vague.  Technically, a user who has a failed in_person_enrollment has also begun in person proofing.

Reply:
> Unscheduled should only care about establishing in-person enrollments right?

I have a n00b question.  If unscheduled only refers to establishing in_person_enrollments, why schedule an enrollment when a user has a pending or establishing in_person_enrollment?  Could this conditional be refactored to only check if the user has an establishing in_person_enrollment?  

The enrollment helper’s `schedule_in_person_enrollment` [returns if the user doesn’t have an establishing in_person_enrollment](https://github.com/18F/identity-idp/blob/main/app/services/usps_in_person_proofing/enrollment_helper.rb#L7-L8),  which makes me think that this refactor wouldn’t actually change the method’s behavior.

Let me know if I’m missing something.